lib/ridgepole/delta.rb

```diff
@@ -191,7 +191,7 @@ def raise_exception(script, org)
       errmsg = lines.with_index.map do |l, i|
         line_num = i + 1
         prefix = line_num == err_num ? '* ' : '  '
-        format("#{prefix}%<line_num>#{digit_number}d: #{l}", line_num: line_num)
+        format("#{prefix}%<line_num>#{digit_number}d: ", line_num: line_num) + l
```

Comment:
Since we are using the format function, how about writing the following?

```suggestion
        format("#{prefix}%<line_num>#{digit_number}d: %<line>s", line_num: line_num, line: l)
```

Reply:
thank you for your advice.
changed 7ca22d4